### PR TITLE
Make start page incredible

### PR DESCRIPTION
**Pull request by Nostix**

### CODE REVIEW

File: app/assets/stylesheets/application.sass [hunk 1]
```diff
@@ -17,32 +17,144 @@
...
+    left: 0
+    opacity: 0
+  5%
+    opacity: 0.0
+  48%
+    opacity: 0.2
+  80%
+    opacity: 0.0
+  100%
+    left: 82%
```
Comment (nicolas-heinig) [inline app/assets/stylesheets/application.sass]: Can you please move these css stuff for the frontend button into a sass partial?

something like `_start_page_button.sass` would be perfect

File: app/assets/stylesheets/application.sass [hunk 2]
```diff
@@ -17,32 +17,144 @@
...
 
 body
+  background: grey
+  font-family: "Arial"
+  background-image: image-url("bg.jpg")
+  background-size: cover
   padding-top: 50px
```
Comment (nicolas-heinig) [inline app/assets/stylesheets/application.sass]: Love it 👍

**Comment by nicolas-heinig** [comment]
I like it! 👍 Can I merge it or is something missing?

**Comment by Nostix** [comment]
Everything seems to be working now!
Ready to merge! 👍

**Pull request merged**